app/src/main/java/com/example/cm/ui/adapters/MeetupDetailedTabAdapter.java

```diff
@@ -0,0 +1,37 @@
+package com.example.cm.ui.adapters;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.viewpager2.adapter.FragmentStateAdapter;
+
+import com.example.cm.data.models.Meetup;
+import com.example.cm.ui.meetup.MeetupDetailed.MeetupDetailedFriendsListFragment;
+
+public class MeetupDetailedTabAdapter extends FragmentStateAdapter {
```

Comment:
Maybe re-structure the Adapter similar to "MeetupTabAdapter" to get rid of hardcoding size as well as returning the correct fragment

Reply:
Not possible because MeetupDetailedFriendsListFragment has a parameter